src/model/repositories/MovieRepository.php

```diff
@@ -134,4 +134,46 @@ public function archiveMovie(int $movieId): void {
             throw new PDOException("Unable to archive movie!");
         }
     }
-}
\ No newline at end of file
+
+    public function restoreMovie(int $movieId): void {

```

Comment:
It's just something minor, but I think it would be a good practice to return 'boolean' instead of 'void', indicating whether the operation was successful. ($query->execute(...) returns a boolean, so you could just very simply return the result of that line